source.go

```diff
@@ -135,6 +147,34 @@ func (s *Source) Teardown(ctx context.Context) error {
 	return nil
 }
 
+func (s *Source) readingAllTables() bool {
+	return len(s.config.Tables) == 1 && s.config.Tables[0] == common.AllTablesWildcard
```

Comment:
A really tiny nitpick: this can be replaced with `s.config.Tables == []string{common.AllTablesWildcard}`